next/components/forms/steps/FinalStep.tsx

```diff
@@ -57,7 +57,7 @@ export const FinalStep = ({
 
   return (
     <div>
-      <h1 className="text-h1-lg-medium">{t('summary')}</h1>
+      <h1 className="text-[48px] font-semibold">{t('summary')}</h1>
```

Comment:
the font should match the typography figma and therefore one of the predefined text-h*-something classes - if it doesn't and such variant does not exist, it should be adjusted in figma not in implementation

please recheck the design against the [foundations file](https://www.figma.com/file/ctDKMhAPIjLUVNNmq430D4/DS-ESBS%3A-Foundations?node-id=65%3A47&t=2wcpn2DaYaXsW5w9-0) and ping relevant people in slack if it does not match